samples/net/ws_echo_server/prj.conf

```diff
@@ -0,0 +1,48 @@
+CONFIG_NETWORKING=y
+CONFIG_NET_UDP=n
+CONFIG_NET_TCP=y
+CONFIG_NET_IPV6=y
+CONFIG_NET_IPV4=y
+#CONFIG_NET_DHCPV4=y
+
+CONFIG_RANDOM_GENERATOR=y
+CONFIG_TEST_RANDOM_GENERATOR=y
+CONFIG_INIT_STACKS=y
+
+CONFIG_NET_LOG=y
+CONFIG_SYS_LOG_NET_LEVEL=1
+CONFIG_SYS_LOG_SHOW_COLOR=y
+CONFIG_NET_STATISTICS=y
+CONFIG_PRINTK=y
+
+CONFIG_NET_PKT_RX_COUNT=16
+CONFIG_NET_PKT_TX_COUNT=16
+CONFIG_NET_BUF_RX_COUNT=16
+CONFIG_NET_BUF_TX_COUNT=16
+CONFIG_NET_CONTEXT_NET_PKT_POOL=y
+
+CONFIG_NET_IF_UNICAST_IPV6_ADDR_COUNT=3
+CONFIG_NET_IF_MCAST_IPV6_ADDR_COUNT=4
+CONFIG_NET_MAX_CONTEXTS=8
+
+CONFIG_NET_SHELL=y
+
+CONFIG_NET_APP=y
+CONFIG_NET_APP_SETTINGS=y
+CONFIG_NET_APP_NEED_IPV6=y
+CONFIG_NET_APP_NEED_IPV4=y
+CONFIG_NET_APP_MY_IPV6_ADDR="2001:db8::1"
+CONFIG_NET_APP_MY_IPV4_ADDR="192.0.2.1"
+
+CONFIG_WS=y
+CONFIG_WS_SERVER=y
+CONFIG_WS_TLS=n
+
+CONFIG_MBEDTLS=y
```

Comment:
Should the MBEDTLS settings be removed from this .conf?

Reply:
This is needed for sha1 and base64 functions that are used even if we do not have TLS supported.

Reply:
Perhaps a comment line in this prj.conf would help then.